### PR TITLE
Add link to MOEA/D blog post in News

### PR DESCRIPTION
**Pull request by y0z**
## Motivation & Description of the changes

This PR adds a link to the MOEA/D blog post in the news section.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,12 +25,12 @@ Optuna can dynamically construct the search spaces for the hyperparameters.
 
 ## :loudspeaker: News
 <!-- TODO: when you add a new line, please delete the oldest line -->
+* **Oct 21, 2024**: We posted [an article](https://medium.com/optuna/an-introduction-to-moea-d-and-examples-of-multi-objective-optimization-comparisons-8630565a4e89) introducing [MOEA/D](https://hub.optuna.org/samplers/moead/) and examples of multi-objective optimization comparisons.
```
Comment (c-bata) [inline README.md:28]: nit: I adjusted the wording as the article includes only one example comparison.

```suggestion
* **Oct 21, 2024**: We posted [an article](https://medium.com/optuna/an-introduction-to-moea-d-and-examples-of-multi-objective-optimization-comparisons-8630565a4e89) introducing [MOEA/D](https://hub.optuna.org/samplers/moead/) and an example comparison with other optimization methods.
```

**Review by c-bata — COMMENTED** [review]
Thank you for your pull request. I left one minor suggestion.

File: README.md (hunk 1 above)
Comment (y0z) [reply README.md]: Updated.

**Review by c-bata — APPROVED** [review]
LGTM!

**Comment by c-bata** [comment]
Let me merge this PR with only one approval since this PR just updates `README.md`.

**Pull request merged**